src/components/LoadingIndicator/LoadingIndicator.tsx

```diff
@@ -0,0 +1,81 @@
+import clsx from 'clsx';
+import React from 'react';
+
+import { Oval } from 'react-loader-spinner';
+import styles from './LoadingIndicator.module.css';
+
+export type Props = {
+  /**
+   * CSS class names that can be appended to the component.
+   */
+  className?: string;
+
+  /**
+   * Layout size of the loader. This affects the overall size and associated
+   * stroke width.
+   */
+  size?: 'sm' | 'md' | 'lg';
+
+  /**
+   * Aria label of the oval. Default is oval-loading. Will be overridden if ariaLabel is passed in props
+   * (Shadowed from OvalProps)
+   *
+   * @see https://mhnpd.github.io/react-loader-spinner/docs/components/oval
+   */
+  ariaLabel?: string;
+
+  /**
+   * Whether the oval is visible. Default is true.
+   * (Shadowed from OvalProps)
+   *
+   * @see https://mhnpd.github.io/react-loader-spinner/docs/components/oval
+   */
+  visible?: boolean;
+};
+
+// Pixel sizes corresponding to EDS size units (pixel = rem * 16)
```

Comment:
I don't understand what the `(pixel = rem * 16)` part means. Are you saying a rem is 16 pixels?

Reply:
Yep that's it. I can actually remove that and let the size comments tell people where to find the value